
<launch>
	<group ns="robot_0">
		<node pkg="ugbots_ros" name="robotnode" type="R1"/>
	</group>
	
	<group ns="robot_1">
		<node pkg="ugbots_ros" name="robotnode" type="R1"/>
	</group>

</launch>
